go.mod

```diff
@@ -3,15 +3,16 @@ module github.com/ONSdigital/dp-observation-api
 go 1.13
 
 require (
-	github.com/ONSdigital/dp-api-clients-go v1.9.0
-	github.com/ONSdigital/dp-graph/v2 v2.0.3
-	github.com/ONSdigital/dp-healthcheck v1.0.3
-	github.com/ONSdigital/dp-rchttp v1.0.0 // indirect
-	github.com/ONSdigital/go-ns v0.0.0-20200205115900-a11716f93bad
+	github.com/ONSdigital/dp-api-clients-go v1.10.1
+	github.com/ONSdigital/dp-authorisation v0.0.0-20200416132558-8ad7b7fd9cee
```

Comment:
Do we need a release for this before merging?

Reply:
Possibly, this is the first application to try and use dp-authorisation library, I will create a v0.1.0 version and update go mod here